prow/config.yaml

```diff
@@ -10684,7 +10684,7 @@ periodics:
       - --bare
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180212-83c830734-master
 
-- cron: "0 0 31 2 *" # manual (set to Feb 31 so it won't trigger) (TODO: automate this when possible)
+- interval: 6h
```

Comment:
Fine for now, but I think it's too frequent in general.

Reply:
Yeah.. I've put the short time just so it runs continuously (at least while I experiment). We can reduce it later.